src/python/src/grpc/_adapter/_blocking_invocation_inline_service_test.py

```diff
@@ -43,4 +43,4 @@ class BlockingInvocationInlineServiceTest(
 
 
 if __name__ == '__main__':
-  unittest.main()
+  unittest.main(verbosity=2)
```

Comment:
Changes the real-time normal test output to be the test names rather than `.`s and `S`s and `E`s.